### PR TITLE
refactor: move admin-config to resource directory and extract common …

### PR DESCRIPTION
**Pull request by CarolineZhang666**
refactor: move admin-config to resource directory and extract common dependencies to pom.xml of shenyu-integrated-test

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-integrated-test/pom.xml [hunk 1]
```diff
@@ -38,45 +38,73 @@
...
-            </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
+        </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
```
Comment (flycash) [inline shenyu-integrated-test/pom.xml:91]: these two dependencies need to specific version.
<version>xxx</version>

File: shenyu-integrated-test/pom.xml [hunk 2]
```diff
@@ -38,45 +38,73 @@
...
-            <dependency>
-                <groupId>org.projectlombok</groupId>
-                <artifactId>lombok</artifactId>
-                <version>1.18.20</version>
-            </dependency>
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-spring-boot-starter-plugin-sign</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```
Comment (flycash) [inline shenyu-integrated-test/pom.xml:74]: please move this to sub pom.xml

**Comment by CarolineZhang666** [comment]
@flycash, please help to review this PR

**Comment by CarolineZhang666** [comment]
@flycash, please help to review this PR

**Review by flycash — CHANGES_REQUESTED** [review]
please rebase to 1 commit

**Comment by CarolineZhang666** [comment]
@flycash, please help to review this PR

**Comment by CarolineZhang666** [comment]
@flycash, please help to review this PR

**Comment by CarolineZhang666** [comment]
@flycash, please help to review this PR

**Pull request closed without merging**